frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -359,9 +639,62 @@ function ReaderBookmark:onShowBookmark()
     end
 
     UIManager:show(self.bookmark_menu)
+    if open_navigator then
+        if #self.bookmarks > 0 then
+            -- open first bookmark in page order:
+            bm_menu:onMenuHold(self.bookmarks[#self.bookmarks])
+        else
+            UIManager:show(InfoMessage:new { text = _('Bookmarks navigator: no bookmarks defined yet') })
```

Comment:
Have we named that thing `Bookmarks navigator` anywhere else?
If not, may be just `'Bookmarks: no bookmark defined yet'`.